MAINTAINERS.yml

```diff
@@ -60,6 +60,28 @@
 #         Like 'files-regex', but any matching files will be excluded from the
 #         area.
 #
+#     file-groups:
+#         A list of groups of files that are treated as a single unit.
+#         This is useful for areas where different collaborators are responsible for
+#         different parts of the area.
+#         Each group should have the following structure:
+#         - name: <group name>
+#           collaborators:
```

Comment:
these collaborators are added in addition to the area collaborators, right? Could maybe be mentioned.